custom/icds_reports/tasks.py

```diff
@@ -256,6 +257,7 @@ def move_ucr_data_into_aggregation_tables(date=None, intervals=2):
                             ).apply_async()
 
             res_awc.get()
+            _bust_awc_cache.delay()
```

Comment:
Why are you changing this, this is not related to eCHIS, please remove this.

Reply:
I don`t know how that change is made... I didn`t touch any other files other than the 8 ones that we are working on...checked all my commits and I was not able to find that change.

Reply:
I think that line of code and others on that file are added after I opened this pull request.  Check this commit
Revision: ac177a07ee1e6a6410996adf0d6f179b7e3afbbd
Author: Cal Ellowitz <calellowitz@gmail.com>
Date: 2/13/2019 5:13:44 PM
Message:
pull cache bust out of main task